nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/NotificationServiceManager.java

```diff
@@ -27,7 +44,9 @@
 import org.apache.nifi.components.PropertyValue;
 import org.apache.nifi.components.ValidationContext;
 import org.apache.nifi.components.ValidationResult;
+import org.apache.nifi.parameter.ParameterLookup;
 import org.apache.nifi.registry.VariableRegistry;
+import org.apache.nifi.security.xml.XmlUtils;
```

Comment:
Now that nifi-bootstrap depends on nifi-security-utils, we'll have to make sure it's in the `lib/bootstrap/` directory in the assembly:

nifi-assembly/src/main/assembly/common.xml line 31 insert:
`<include>nifi-security-utils</include>`

Reply:
Looks like there's a better solution by just making it available via nifi-framework-authorization, will try with the latest